tools/cldr-code/src/test/java/org/unicode/cldr/test/TestEnInheritance.java

```diff
@@ -0,0 +1,48 @@
+package org.unicode.cldr.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.logging.Logger;
+import org.junit.jupiter.api.Test;
+import org.unicode.cldr.util.*;
+
+/**
+ * Check modern coverage inherited values in en that are marked with up arrows or are blank; all
+ * values should be explicit in en
+ */
+public class TestEnInheritance {
+    private static final Logger logger = Logger.getLogger(TestEnInheritance.class.getName());
+
+    private static final String LOCALE_ID = "en";
+
+    @Test
+    void testEn() {
+        final CLDRConfig config = CLDRConfig.getInstance();
+        final CLDRFile cldrFile = config.getCLDRFile(LOCALE_ID, false);
```

Comment:
Also, you need to check on this. We need to hit not only the main directory, but also the annotations directory. I think Factory getCommonAndSeedAndMainAndAnnotationsFactory() is the safest.